esp-hal/src/i2s.rs

```diff
@@ -73,6 +75,9 @@
 //! }
 //! # }
 //! ```
+//! 
+//! ## Implementation State
+//! - Only master mode is supported.
```

Comment:
IIRC we also only implement the TDM Phillips audio standard, which should probably be noted here as well